src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+  const status = 'Next Player: ' + (isXNext ? 'X' : 'O')
+
+  /**
+   * based on the range value, sets the game to history[n]
+   * @param n
+   */
+  function jumpTo(n: number) {
+    setIsXNext(n % 2 === 0)
+    const newHist = history.slice(0, n + 1)
+    setHistory(newHist)
+  }
+
+  /**
+   * pointer function for handling click events from square component
+   * adds current board state to history[] state
+   * @param squares
+   */
+  function update(squares: string[]) {
+    const updatedHist = [...history, squares]
+    setHistory(updatedHist)
```

Comment:
```suggestion
    setHistory([...history, squares])
```

I'd just combine this into a one-liner